runtime/bundles/org.eclipse.e4.core.services/src/org/eclipse/e4/core/services/contributions/IContributionFactory.java

```diff
@@ -26,4 +28,8 @@ public interface IContributionFactory {
 
 	Bundle getBundle(String uriString);
 
+	default boolean isEnabled(String uriString) {
```

Comment:
I wonder why this added API did not need a `@since` javadoc

Reply:
Because it is only preliminary (non public) API

Reply:
Thanks, i did not have in mind x-friends declares provisional APIs